keyboards/handwired/mingwahlu/mingnumpad/readme.md

```diff
@@ -0,0 +1,24 @@
+# mingnumpad
+
+![mingnumpad assembled](https://i.imgur.com/50zzoPy.jpg)
+![mingnumpad insides](https://i.imgur.com/RZ7BMlZ.jpg)
+
+
+*A short description of the keyboard/project*
+This is a handwired numpad using an Arduino Pro Micro and a 3D printed case from thingiverse.
+
+* Keyboard Maintainer: [mingwahlu](https://github.com/tan00060)
+
+* Hardware Supported: Ardiuno Pro Micro ATmega32U4: [ArdiunoProMicro](https://www.amazon.ca/KeeYees-ATmega32U4-Development-Microcontroller-Bootloader/dp/B07FXCTVQP/ref=sr_1_3?dchild=1&keywords=arduino+pro+micro&qid=1598072684&sr=8-3)
+
+* Hardware Availability: 3D Print of the numpad case: [Numpad](https://www.thingiverse.com/thing:3393103)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make make handwired/mingwahlu/mingnumpad:via
+
+Flashing example for this keyboard:
+
+    make make handwired/mingwahlu/mingnumpad:via:avrdude
```

Comment:
```suggestion
    make make handwired/mingwahlu/mingnumpad:via:flash
```

Reply:
Have updated to  make handwired/mingwahlu/mingnumpad:via:flash instead of make handwired/mingwahlu/mingnumpad:via:avrdude